Public/Src/Utilities/Utilities/FileArtifact.cs

```diff
@@ -237,8 +225,22 @@ private string ToDebuggerDisplay()
                     : Path.ToString(owner);
             }
 
-            return I($"{{{path} ({annotation})}}");
+            return I($"{{{path} ({GetAnnotation()})}}");
         }
+
+        private string GetAnnotation()
+        {
+            switch (m_rewriteCount)
+            {
+                case 0:
+                    return "source";
+                case 1:
+                    return "output";
+                default:
+                    return "rewrite:" + m_rewriteCount.ToString(CultureInfo.InvariantCulture);

```

Comment:
> m_rewriteCount.ToString(CultureInfo.InvariantCulture) [](start = 39, length = 54)

why do we need Inv Culture here? m_rewriteCount is an int.

nit: `$"rewrite: {m_rewriteCount}"` or `I($"rewrite: {m_rewriteCount})"`